spyder/app/mainwindow.py

```diff
@@ -94,11 +94,16 @@
 
 #==============================================================================
 # Proper high DPI scaling is available in Qt >= 5.6.0. This attibute must
-# be set before creating the application
+# be set before creating the application. 
 #==============================================================================
+from spyder.config.main import CONF

```

Comment:
`CONF` is also imported below (around line `140`). Please remove from there to not import it twice :-)